pkg/controllers/allocation/controller.go

```diff
@@ -66,13 +86,15 @@ func NewController(kubeClient client.Client, coreV1Client corev1.CoreV1Interface
 // Reconcile executes an allocation control loop for the resource
 func (c *Controller) Reconcile(ctx context.Context, object client.Object) (reconcile.Result, error) {
 	provisioner := object.(*v1alpha2.Provisioner)
+	c.batcher.Wait(fmt.Sprintf("%s/%s", provisioner.Name, provisioner.Namespace))
+
 	// 1. Filter pods
 	pods, err := c.filter.GetProvisionablePods(ctx, provisioner)
 	if err != nil {
 		return reconcile.Result{}, fmt.Errorf("filtering pods, %w", err)
 	}
 	if len(pods) == 0 {
-		return reconcile.Result{RequeueAfter: c.Interval()}, nil
+		return reconcile.Result{}, nil
```

Comment:
If a pod is unschedulable for some reason, will we *ever* attempt to reschedule it? What if there's an ICE?

Reply:
In the case of capacity failing to come up for any reason, the Reconcile returns an error and the controller will retry based on the work-queue options `ExponentialFailureRateLimiter`. It looks like we're not returning an error in the aws cloudprovider `Create` if partial capacity is fulfilled.  I think that needs to be changed and looked at more closely.

Reply:
We currently have a semantic of "continue after" that needs to be address with this change. Alternatively, you can keep an interval in (in addition to batching) and address it in a separate PR.

Reply:
Yeah I'll probably add back a requeue and address in a separate PR